Sources/Purchasing/StoreKitAbstractions/EncodedAppleReceipt.swift

```diff
@@ -29,7 +30,13 @@ extension EncodedAppleReceipt {
         case .jws(let jws):
             return jws
         case .receipt(let data):
-            return data.asFetchToken
+            return data.base64EncodedString()
+        case .sk2receipt(let receipt):
+            do {
+                return try receipt.jsonEncodedData.base64EncodedString()
+            } catch {
+                return ""
```

Comment:
Let's log an error here.

Reply:
Added a log here and in `PostReceiptDataOperation.swift` when there's an error encoding an SK2 receipt